src/resources/users/dto/user.dto.ts

```diff
@@ -0,0 +1,55 @@
+import { ApiProperty } from '@nestjs/swagger';
+import {
+	ArrayMaxSize,
+	IsArray,
+	IsEnum,
+	IsNumber,
+	IsOptional,
+	IsString,
+} from 'class-validator';
+import { businessEntityTypeEnum, userTypeEnum } from 'src/drizzle/schema/users';
+
+export class UserDto {
```

Comment:
Should Be UserEntity and under entities directory `users/entities/user.entity.ts`